modules/core/src/main/java/org/apache/ignite/configuration/ClientConfiguration.java

```diff
@@ -529,11 +531,17 @@ public ClientConfiguration setTransactionConfiguration(ClientTransactionConfigur
     }
 
     /**
+     * Partition awareness functionality helps to avoid an additional network hop in the following scenarios:
+     * <ul>
+     *     <li>1. Single-key operations API, like put(), get(), etc. However, the functionality has no effect on those
+     *     operations within explicit transactions {@code ClientTransactions#txStart()}.</li>
+     *     <li>2. {@link ScanQuery#setPartition(Integer)} and {@link IndexQuery#setPartition(Integer)} accept a
+     *     partition number as a parameter with which the query is routed to a particular server node that stores
+     *     the requested data.</li>
+     * </ul>
      * @return A value indicating whether partition awareness should be enabled.
-     * <p>
-     * Default is {@code true}: client sends requests directly to the primary node for the given cache key.
-     * To do so, connection is established to every known server node.
-     * <p>
+     * <p>Default is {@code true}: client sends requests directly to the primary node for the given cache key.
```

Comment:
Let's move this part at the beginning of javadoc. @return should be the last line in the doc:
```
<p>Default is  ....
When {@code false}...
<p>
Partition awareness functionality...

```

Reply:
thank you, I will correct